docs/user-guide/src/bmo/automatic_secure_boot.md

```diff
@@ -0,0 +1,8 @@
+# Automatic secure boot
+
+The automatic secure boot feature allows enabling and disabling UEFI secure boot on and off when provisioning a host.
+
+why we need it?
+We need automatic secure boot when provisoing a host with high security requirements 
+how to use it?
+To enable Automatic secure boot we need to specify the value `UEFISecureBoot` for `bootmode` in the bmh custom resource 
```

Comment:
`bootMode`

Somewhere here we need to explain that:

1. It only works with some drivers (I think only redfish-based, definitely not IPMI).
2. Secure boot is enabled before booting into the deployed instance and disabled when the ramdisk is running and on tear down.

Reply:
Is the secure boot disabled on deprovisioning or once the instance booted ? should I specify ramdisk image here ?

Reply:
I have added the second point as a note to the user as is.